File: docs/code_examples/point3d_simple.cpp
// Log some very simple points.

#include <rerun.hpp>

namespace rr = rerun;

int main() {
    auto rec = rr::RecordingStream("rerun_example_points3d_simple");
    rec.connect("127.0.0.1:9876").throw_on_failure();

    rec.log("points", rr::Points3D({{0.0f, 0.0f, 0.0f}, {1.0f, 1.0f, 1.0f}}));
}

File: docs/code_examples/point3d_simple.rs
//! Log some very simple points.

use rerun::{archetypes::Points3D, RecordingStreamBuilder};

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let (rec, storage) = RecordingStreamBuilder::new("rerun_example_points3d_simple").memory()?;

    rec.log("points", &Points3D::new([(0.0, 0.0, 0.0), (1.0, 1.0, 1.0)]))?;

    rerun::native_viewer::show(storage.take())?;
    Ok(())
}
